### PR TITLE
Added Entra portal reference

### PR DESCRIPTION
**Pull request by miguel-s-ferreira**
Added the new Entra portal available to centralize identity and governance features on cloud

### CODE REVIEW

**Comment by denisebmsft** [comment]
Hello @miguel-s-ferreira. I need to verify whether we can push this live yet.

File: microsoft-365/security/defender/portals.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ While these portals are not specifically for managing security, they support var
 
 | Portal name | Description | Link |
 |---|---|---|
+| Entra portal | Access and administer the [Microsoft Entra](/entra) family, to protect your business with decentralized identity, identity protection, governance and more in a multi-cloud environment | [entra.microsoft.com](https://entra.microsoft.com/) |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/portals.md]:
```suggestion
| Entra portal | Access and administer the [Microsoft Entra](/entra) family to protect your business with decentralized identity, identity protection, governance, and more, in a multi-cloud environment | [entra.microsoft.com](https://entra.microsoft.com/) |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @miguel-s-ferreira, kindly commit the suggested minor changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @miguel-s-ferreira

**Review by denisebmsft — APPROVED** [review]

**Pull request merged**